disnake/app_commands.py

```diff
@@ -352,6 +372,8 @@ def add_option(
         autocomplete: bool = False,
         min_value: float = None,
         max_value: float = None,
+        min_length: int = None,
+        max_length: int = None,
```

Comment:
Similar change missing for `SlashCommand.add_option`

Reply:
https://github.com/DisnakeDev/disnake/pull/593/commits/714cdf1ef34624bbeeed9cce2e6e3a035a687c09